pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -370,6 +372,22 @@ func (r *resourceManager) emptyService() *corev1.Service {
 func (r *resourceManager) ensureDeployment(ctx context.Context) error {
 	deployment := r.emptyDeployment()
 
+	// TODO(rfranzke): Remove this special handling when we only support seed clusters of at least K8s 1.20.
+	//                 Then we can use the 'kube-root-ca.crt' configmap to get access to the CA cert.
+	var rootCAVolumeSourceName string
```

Comment:
just a suggestion: 
a function like:
`getPathToRootCAFromServiceAccount`
instead of the `{}`
makes functions like ensureDeployment smaller in length